tmp/lanelet2_extension/include/lanelet2_extension/projection/transverse_mercator_projector.hpp

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// NOLINTBEGIN(readability-identifier-naming)
+
+#ifndef LANELET2_EXTENSION__PROJECTION__TRANSVERSE_MERCATOR_PROJECTOR_HPP_
+#define LANELET2_EXTENSION__PROJECTION__TRANSVERSE_MERCATOR_PROJECTOR_HPP_
+
+// NOLINTBEGIN(readability-identifier-naming)
+
+#include <GeographicLib/MGRS.hpp>
```

Comment:
Are there any reasons for including MGRS.hpp?

Reply:
Thanks, not particularly, just forgot to remote. Reflected here: https://github.com/autowarefoundation/autoware_common/pull/191/commits/1a085a3b6b175165e10a851fca53611b84cc8122